layers/meta-balena-beaglebone/recipes-kernel/kernel-selftest/kernel-selftest.bbappend

```diff
@@ -0,0 +1,7 @@
+
+PACKAGECONFIG:append:aarch64 = " watchdog"
+TEST_LIST:append:aarch64 = "\
```

Comment:
How is this supposed to be used?

Reply:
This builds a `test-watchdog` package in the deploy directory. This helps to quickly identify if the current version of the kernel has a problem with the hardware watchdog (this is still an open issue, btw).